frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts

```diff
@@ -347,6 +347,9 @@ export const sessionRecordingsPlaylistLogic = kea<sessionRecordingsPlaylistLogic
         ],
         showAdvancedFilters: [
             addedAdvancedFilters(props.filters, getDefaultFilters(props.personUUID)),
+            {
+                persist: true,
```

Comment:
I don't think this works. If we persist it then it will be the new default, but the logic is trying to determine whether to show advanced options based on the filters set.

Reply:
Ah, interestingly the `hasAdvancedFilters` check is broken in prod 🤣 -> into the 🕳️

Reply:
OK, I think I got to the bottom of the rabbit hole